sources/packages/backend/libs/services/src/sfas/sfas-application.service.ts

```diff
@@ -1,7 +1,9 @@
 import { Injectable } from "@nestjs/common";
-import { DataSource, Brackets } from "typeorm";
+import { DataSource, Brackets, MoreThanOrEqual } from "typeorm";
 import { DataModelService, SFASApplication } from "@sims/sims-db";
 import { LoggerService, InjectLogger } from "@sims/utilities/logger";
+import { MAX_MSFAA_VALID_DAYS } from "@sims/utilities";
+import { SFASSignedMSFAA } from "@sims/services/sfas/sfas-individual.model";
```

Comment:
Please add `SFASSignedMSFAA` file to the index in the same file and consume from there.
```ts
import { SFASSignedMSFAA } from ".";
```